docs/server/commands.rst

```diff
@@ -287,6 +287,15 @@ Accepts the :option:`--modified-since` parameter to list only those projects
 modified since the revision given by :djadmin:`revision`.
 
 
+.. django-admin:: contributors
+
+contributors
+^^^^^^^^^^^^
+
+Lists the contributors to a language, project or overall and the amount
+of contributions they have.
```

Comment:
Noticed that you added `--from-revision` which is not explained here.

Also more detailed explanation on what result you will get when passing this or other options will most likely be appreciated by people reading the docs.